### ISSUE TITLE
Cohesive structure on the Nisnevich $(\infty, 1)$-topos of Sch/S?

### ISSUE THREAD

**Issue by xuanruiqi** [felixwellen/synthetic-zariski#33]
This is a sort of continuation to #18.

Does the Nisnevich $(\infty, 1)$-topos on $\mathbf{Sch}/S$ admit a useful cohesive structure?

The [nLab page on motivic homotopy theory](https://ncatlab.org/nlab/show/motivic+homotopy+theory) suggests that if one is working in the right topos (i.e., Nisnevich), then localizing at $\mathbb{A}^1$ will appropriately give one the motivic homotopy theory. Now, I'm probably interpreting it very wrong, but it seems to follow that if one is working in the Nisnevich topos (as one should be), _then_ localization at $\mathbb{A}^1$ will give one the shape modality one wants, giving the higher Nisnevich topos the cohesive structure one wants.

I wonder what one can make out of this.

**Comment by xuanruiqi** [comment]
By the way, just a suggestion to @felixwellen : for this kind of open-ended questions, GitHub discussions seems to be a more appropriate tool than issues. Maybe you could enable discussions in the repo settings?

**Comment by hmoeneclaey** [comment]
Hi,
I think motivic homotopy type do not form a higher topos (Remark 3.5 in https://arxiv.org/pdf/1008.4915.pdf). Wouldn't such a presentation contradicts this? Slide 25 in http://home.sandiego.edu/~shulman/papers/cmu2019a.pdf seems to imply this, but I might be misunderstanding.

**Comment by hmoeneclaey** [comment]
It of course depends on what you mean by useful cohesive structure...

**Comment by felixwellen** [comment]
It has the same limitations as discussed in #18: The flat modality cannot be as good as in cohesive HoTT, because that would imply the localization is a topos. This is even more direct for a sharp-modality, which, by being right adjoint, would be a lex monadic modality.
It could still be the case that there is some flat modality with less/different properties than the one from cohesive HoTT, but so far I could only think of applications which need the lexness it cannot have. 
Also mind that in general, the ideas behind cohesion don't go too well with algebraic geometry. In all examples I have looked at, cohesion is about things which are made from one kind of point, but in algebraic geometry you usually have many different kinds of points (e.g. Spec of extensions of some base field). Furthermore, in cohesion, things are "made from contractible blobs", which more technically means local contractibility. I don't remember when and how that fails in algebraic geometry, but there is a nice, readable conversation on that [here](https://golem.ph.utexas.edu/category/2020/03/pyknoticity_versus_cohesivenes.html)

**Comment by xuanruiqi** [comment]
Yeah, obviously I was a big misguided here. On a more basic level, I think this is related to some fundamental problems with $\mathbb{A}^1$-homotopy theory (coincidentally from Keyao Peng iirc) - many things that ought to be $\mathbb{A}^1$-invariant are in fact not, so one can expect lots of problems with the very idea of $\mathbb{A}^1$-invariance.

**Comment by xuanruiqi** [comment]
The link you sent is interesting - I need some time to read through that though. But you are right that "cohesive" does mean "topological space-like" in many senses, and we already know very well that fails pretty badly on algebraic objects.

**Comment by xuanruiqi** [comment]
> are made from contractible blobs

Yeah, I guess when I was talking about topological spaces, I was thinking of "nice" spaces.

> covering theory like it would be phrased in cohesion

Doesn't covering require only the existence of a shape modality, so not full cohesion? Sure we can do everything that can be done in cohesive HoTT that does not require the flat or sharp modalities (that would be a tautological statement). I guess I was trying to see whether there is a more general setting in which we could look at $\mathbb{A}^1$-homotopy theory, but maybe it is right that we can't because it's still full of quirks.

**Comment by xuanruiqi** [comment]
BTW I found [this answer](https://mathoverflow.net/questions/356618/what-is-the-precise-relationship-between-pyknoticity-and-cohesiveness) written by Peter Scholze - indeed the problem comes from schemes not being locally contractible. He seems to suggest, though, that if one switches to the pro-etale (i.e. condensed) setting, things might start to work...

**Comment by xuanruiqi** [comment]
Some [follow up work](https://1429cecd-24a0-4223-8b7c-1ebf47aa12e2.filesusr.com/ugd/8e912a_fe1c2f8e90094d0fa065fb8129687963.pdf). I think this suggests that we get cohesion only if we move into the condensed world - I don't know what that will imply at the time being...

**Comment by felixwellen** [comment]
> Doesn't covering require only the existence of a shape modality, so not full cohesion?

Yes, a shape modality is a "relevant part" of cohesion I would say, even though it is just a modality ;-)
Having a flat modality makes it easier to show that things are covering spaces though.

**Comment by xuanruiqi** [comment]
Judging from the thesis I sent in the previous comment - cohesiveness is generally not a useful concept in an algebraic setting. On the other hand, _fracturedness_, developed by Lurie in the context of spectral algebraic geometry and used successfully by Clough (2023) to study differentiable sheaves, seem to be a much more useful concept.

There is currently no synthetic account of fracturedness, but I expect to be useful in SAG once it is developed. It can't fix the problem that motivic spaces do not form a topos per se, but it should be useful in _some_ way.

**Comment by xuanruiqi** [comment]
I suggest leaving this issue open for further discussion, though. Maybe change the title?

**Comment by felixwellen** [comment]
The (related) differential hexagon has been treated synthetically:
https://arxiv.org/abs/2106.15390
